python/rmm/_lib/torch_allocator.pyx

```diff
@@ -0,0 +1,22 @@
+from cuda.ccudart cimport cudaStream_t
+from libc.stdint cimport uintptr_t
+from libc.stdio cimport printf
+
+from rmm._lib.memory_resource cimport device_memory_resource
+
+
+cdef extern from "rmm/mr/device/per_device_resource.hpp" namespace "rmm" nogil:
+    cdef device_memory_resource* get_current_device_resource \
```

Comment:
Should this use `from rmm._lib.memory_resource cimport get_current_device_resource` like in `device_buffer.pyx`?

Reply:
I added a new `per_device_resource.pxd` file from which we can `cimport` the function.

Note that we cannot get away with declaring `get_current_device_resource` in `memory_resource.pxd`, because `memory_resource` exports a `cpdef` function also named `get_current_device_resource` that is a wrapper around the C++ function.